docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -116,7 +117,8 @@ and Beats as soon as possible to take advantage of performance improvements
 and other enhancements.
 
 [discrete]
-=== Upgrading to 6.3
+=== Upgrading from 5.6 - 6.2
+
```

Comment:
The following reference to "rolling upgrades to 6.3" should be "rolling upgrades to {version}"